scripts/start

```diff
@@ -12,10 +17,10 @@ cd "${projectRoot}"
 docker-compose down
 docker-compose -f docker-compose.migrate.yaml down
 # Ensure container images are up to date
-docker-compose -f docker-compose.migrate.yaml run database-up
+docker-compose -f docker-compose.migrate.yaml run --rm database-up
```

Comment:
What's this doing?

See it's missing from other repos' `start` scripts

Reply:
The docker run command will result in that script running in a container.  At the end you end up with a stopped container.  Every time you run the start script you'll get another container so get a build up over time.

So that command removes the container once it has successfully completed it's job.